sr_gui_joint_slider/src/sr_gui_joint_slider/joint_slider.py

```diff
@@ -102,23 +103,11 @@ def __init__(self, context):
 
         self._widget.reloadButton.setEnabled(True)
 
-        self.hand_prefix = self._get_hand_prefix()
+        self.hand_prefix = get_hand_prefix()
         self._widget.joint_name_filter_edit.setText(self.hand_prefix)
 
         self.on_reload_button_cicked_()
-        self._widget.information_btn.clicked.connect(self.display_information)
-
-    def _get_hand_prefix(self):
```

Comment:
I thought we said we would make them static methods.
Having it a separate function in this module seems less tidy, since no one is going to import this module in order to reuse this function.

Reply:
This PR was done before we agreed on common solutions for these problems but made them now static as suggested.